### PR TITLE
@kanaabe deprecate author href

### PR DESCRIPTION
**Pull request by 1aurabrown**
was seeing some weird urls

### CODE REVIEW

**Comment by kanaabe** [comment]
now that we don't have user profiles, I think it's unnecessary to display this field at all. curious, where is this currently being used?

**Comment by 1aurabrown** [comment]
links on the artist's articles tab, for each article the author's name
links to the href. i've actually only see this linking to /editorial
though, in practice. lmk if it should be killed entirely.

On Thu, Jun 23, 2016 at 9:51 PM, Kana Abe <notifications@github.com> wrote:

> now that we don't have user profiles, I think it's unnecessary to display
> this field at all. curious, where is this currently being used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/artsy/metaphysics/pull/331#issuecomment-228163901>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADBlTnSwx5Ff0f7GR-u59pTAGEWUWpDaks5qOuPMgaJpZM4I9KUR>
> .
>



-- 

[A.]


Laura Brown

Artsy <http://artsy.net/>

laura@artsy.net <caroline@artsy.net>

**Comment by alloy** [comment]
We use it in Eigen: https://github.com/artsy/emission/blob/448a22f0c5d833e59ef84f05d368dd01a0e2f6c8/lib/components/artist/articles/article.js#L20

We also had to guard against `null` there, although in our case the article really doesn’t have an author: https://www.artsy.net/article/yellow-peril-caleb-cain-marcus.

**Comment by alloy** [comment]
Actually we do not use the href at all, just the name :)

So from our end, go ahead and kill it 👍

**Comment by 1aurabrown** [comment]
@kanaabe bump

**Comment by kanaabe** [comment]
whoops, missed this. mergin

File: schema/author.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const AuthorType = new GraphQLObjectType({
     href: {
       type: GraphQLString,
       resolve: ({ profile_handle }) => `/${profile_handle}`,
+      deprecationReason: "Profiles and thus artist hrefs don't exist anymore",
```
Comment (alloy) [inline schema/author.js]: @1aurabrown Was that supposed to be ‘author’ instead of ‘artist’?

Comment (alloy) [inline schema/author.js]: @1aurabrown Assuming this is the case, I’ve added this commit to my PR https://github.com/artsy/metaphysics/pull/313/commits/a62db2d449935bc2007b342d92acae17ec431e7d

**Pull request merged**